CHANGELOG.rst

```diff
@@ -2,6 +2,15 @@
 Changelog for package maliput
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.7.0 (2025-08-27)
+------------------
+* Rules related errors. (`#667 <https://github.com/maliput/maliput/issues/667>`_)
+* Downgrade logging message. (`#671 <https://github.com/maliput/maliput/issues/671>`_)
+* Make a template for Throw method. (`#668 <https://github.com/maliput/maliput/issues/668>`_)
+* Maliput throw new errors. (`#665 <https://github.com/maliput/maliput/issues/665>`_)
+  Co-authored-by: Franco Cipollone <53065142+francocipollone@users.noreply.github.com>
+* Contributors: Franco Cipollone, Santiago Lopez
+
```

Comment:
Have you done git pull in main before running this? I think one is missing

Reply:
Sorry, I just merged this: https://github.com/maliput/maliput/pull/670

Reply:
Aghh, I thought I did. Must have been in malidrive.

Reply:
Done now.